Schemas/CampaignSchema.ts

```diff
@@ -0,0 +1,57 @@
+import { CampaignCustomizationModel } from "@/Models/CampaignModel";
+import { MutableRefObject } from "react";
+import { z } from "zod";
+export const CampaignSchema = z.object({
+  Name: z.string().min(1, "Name is required"),
+  CompanyName: z.string().min(1, "Company name is required"),
+  ContactNumber: z.string().length(10, "Contact number must be 10 digits").regex(/^\d+$/, "Contact Number must contain only digits."),
+  CompanyEmailId: z.string().email("Invalid email address"),
+  Comments: z.string().min(1, "Comments are required!"),
+});
```

Comment:
Move all Text messages to seperate constant file and define variables and access those variables here. do not hardcode anything